rnote-ui/data/ui/workspacebrowser.ui

```diff
@@ -153,5 +175,17 @@
         </child>
       </object>
     </child>
+    <object class="GtkPopoverMenu" id="workspace_button_popover">
+      <property name="name">workspace_button_popover</property>
+      <property name="menu-model">filerow_menu_model</property>
+      <menu id="filerow_menu_model">
```

Comment:
the menu should be renamed to `workspace_dir_menu_model` or something like that, indicating that the menu is for actions operating in the workspace directory.